app/constants/capacity-type.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
I would move this to a constants directory in the `data` folder as it should only be used at the data access layer.